example/plugin-system/custom-plugin/CursorCustomPlugin.js

```diff
@@ -0,0 +1,72 @@
+import CursorPlugin from '../../../src/plugin/cursor.js';
+
+export default class CursorCustomPlugin extends CursorPlugin {
+    /**
+     * Cursor plugin definition factory
```

Comment:
can you change the description of the plugin and explain it's a custom plugin that extends the cursor plugin, and also describe what it does different from cursor plugin?

Reply:
awesome thanks!